Frameworks/include/CppWinRTHelpers.h

```diff
@@ -0,0 +1,139 @@
+//******************************************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#pragma once
+
+#import <Foundation/Foundation.h>
+#import "StringHelpers.h"
+
```

Comment:
We've been presenting projections as the way for our clients to interact with WinRT. I wouldn't want to muddy the waters and start talking about C++/WinRT as an alternative without a very deliberate decision from team leads.

---
In reply to: [109248642](https://github.com/Microsoft/WinObjC/pull/2382#discussion_r109248642) [](ancestors = 109248642)